SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -77,7 +77,7 @@ void FLocalDeploymentManager::Init(FString RuntimeIPToExpose)
 			TryStopSpatialService();
 			// Start spatial service in the current project if spatial networking is enabled
 
-			if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
+			if (SpatialEngineHelper::UsesSpatialNetworking())
```

Comment:
This seems a bit strange. If this PR wants to move the `bSpatialNetworking` flag why not put it in the `SpatialGDKServicesSettings` module which can be used by all other modules (including this one).

Reply:
Also how is `SpatialEngineHelper` being imported?

Reply:
Ignore the `SpatialGDKServicesSetting` comment